mappings/net/minecraft/world/chunk/ImmutableChunk.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2821 net/minecraft/world/chunk/ReadOnlyChunk
+CLASS net/minecraft/class_2821 net/minecraft/world/chunk/ImmutableChunk
```

Comment:
Can the backing chunk still be modified in any way? For example,
```java
Object[] arr = new Object[3];
List<Object> list = Collections.unmodifiableList(Arrays.asList(arr));
arr[0] = Integer.valueOf(8);
```
In this example, `list` is unmodifiable/readonly but not immutable, since its backing array can still change. Is this the case for the backing chunk? If yes, the class should not be named `Immutable`